app/adminPage.html

```diff
@@ -35,6 +36,8 @@
     <script src="js/questionCount.js" defer></script>
     <script src="js/newTest.js" defer></script>
     <script src="js/sendData.js" defer></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.min.js" defer></script>
```

Comment:
using a type here but not using a type in other scripts not using it?

Reply:
changed!